src/olympia/accounts/views.py

```diff
@@ -162,7 +162,8 @@ def parse_next_path(state_parts):
         # but it only cares if there are too few so add 4 of them.
         encoded_path = state_parts[1] + '===='
         try:
-            next_path = base64.urlsafe_b64decode(str(encoded_path))
+            next_path = (
+                base64.urlsafe_b64decode(str(encoded_path)).decode('utf-8'))
```

Comment:
I just replaced the `str` call with `django.utils.encoding.force_bytes`, is that what you were thinking?